cli/cmd/root.go

```diff
@@ -118,25 +98,13 @@ func NewCmdRoot() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&logFileName, "log-file-name", "cloudquery.log", "Log filename")
 
 	// Telemtry (analytics) flags
-	cmd.PersistentFlags().Bool("no-telemetry", false, "disable telemetry collection")
+	cmd.PersistentFlags().String("telemetry-level", "all", "Telemetry level (none, errors, all)")
```

Comment:
What's the use case to have multiple levels? Maybe we can keep it as a boolean for now, due to:
1. Renaming the flag is a breaking change, so better to do it in a backwards compatible
2. I think it will be hard to explain the differences in the levels as it's mostly an internal implementation

Reply:
1. I don't think anyone is using this flag since we first released it last week. This was specifcally asked by the team here - https://github.com/cloudquery/plugin-sdk/issues/128 (I think by you :) )
2. I don't mind splitting those we just need to decide. I merged it because it was requested.

Reply:
Ah, I think we meant to just not call the flag `sentry` and use a non implementation detail named flag like `telemetry`